### PR TITLE
Updated and removed redundant scripts

### LINKED ISSUE #1418: Audit of npm scripts
**Is your feature request related to a problem? Please describe.**
In simorgh there is currently 44 npm scripts, these can be used in one of multiple of the following:
- local dev
- makefile
- jenkins (simorgh)
- travis
- husky checks
- jenkins (infrastructure)
- spartacus (in the future)

**Describe the solution you'd like**
~We should document the purpose of each script for clarity.~ 
Simplify the scripts and naming them better so that they are self-explanatory so that we do not need to maintain documentation for scripts

**Describe alternatives you've considered**
~We could prioritise which scripts to document first rather than doing all 44 in one go.~

**Testing notes**
All the npm scripts available in `package.json` should run fine on CI and locally.

Dev insight: This will be documentation so no manual testing. However, a tester should be able to read through the documentation and understand each 

**Additional context**
I raised this after getting frustrated with `test:lighthouse` and `test:lighthouse:ci`
https://github.com/bbc/simorgh/blob/latest/package.json

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by hotinglok**
### NEEDS TO MERGE WITH https://github.com/bbc/simorgh-infrastructure/pull/666

Reverts bbc/simorgh#3349

Resolves #1418 

**Overall change:**
Reduced overall number of scripts from 44 to 24 through removing redundant scripts and merging scripts together.

**Code changes:**
Replaced/Merged into to other scripts
- ```cypress:proxy``` merged into ```cypress```
- ```test:ampvalidate:ci``` merged into ```test```
- ```build:ci``` moved into ```MakeFile``` (simorgh infrastructure pr: https://github.com/bbc/simorgh-infrastructure/pull/666 )
- ```lighthouse:article``` and ```lighthouse:frontpage``` merged into ```lighthouse```
- ```webpack:prod``` merged into ```build```, ```build:test```, and ```build:live```
- ```forced-package-lock-https``` made into script in /scripts

Removals
- ```test:e2e:local_env/test_env/live_env``` 
Use ```CYPRESS_APP_ENV npm run cypress``` instead.
- ```updateSnapshots```  
Use ```-- -u``` instead.
- ```speculate``` 
Use ```npx speculate``` instead.
- ```audit:ci```
- ```dataValidate```
- ```dataValidate:debug```
- ```killStorybook```
- ```test:ampvalidate``` removed as ```killApp``` does not run on jenkins
- ```test:accessibility ```
- ```test:chromatic```
- ```test:e2e:interactive```

Renamed scripts
- ```test:prod:ci``` renamed as ```test:ci```

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -5,54 +5,29 @@
...
+    "test:chromatic": "chromatic test run --build-script-name build:storybook || true",
+    "test:ci": "npm run start & npm install --no-save cypress@latest bbc-a11y@latest npm-run-all@latest puppeteer@latest audit-ci@latest && run-p cypress test:puppeteer && run-p bbcA11y lighthouse && audit-ci --low",
     "test:dependencies": "npx dependency-check@3.4.1 ./package.json",
-    "test:prod:ci": "npm run start & npm install --no-save cypress@latest bbc-a11y@latest lighthouse@latest npm-run-all@latest puppeteer@latest && run-p cypress test:puppeteer && run-p bbcA11y lighthouse",
-    "test:prod:travis": "CYPRESS_SKIP_EU=true npm run test:prod:ci",
-    "test:e2e": "npm run killApp && npm run build && run-p --race start cypress",
-    "test:e2e:local_env": "CYPRESS_APP_ENV=local npm run test:e2e",
-    "test:e2e:test_env": "CYPRESS_APP_ENV=test npm run cypress",
-    "test:e2e:live_env": "CYPRESS_APP_ENV=live npm run cypress",
-    "test:e2e:interactive": "npm run killApp && npm run build && run-p --race start cypress:interactive",
```
Comment (jamesbhobbs) [inline package.json:44]: I think we should put this back

File: package.json [hunk 2]
```diff
@@ -5,54 +5,29 @@
...
     "start": "NODE_ENV=production node build/server.js",
     "storybook": "start-storybook -p 9001 -c .storybook -h localhost.bbc.com",
-    "storybook:ci": "start-storybook -p 9001 -c .storybook --ci",
-    "test": "npm run test:lint && npm run dataValidate && npm run test:dependencies && npm run test:unit && npm run test:ampValidate:ci",
-    "test:accessibility": "npm run killApp && npm run build && run-p --race start bbcA11y",
-    "test:ampValidate": "npm run killApp && npm run build -- --hide-modules --colors && run-p --race start amp:validate",
-    "test:ampValidate:ci": "npm run build -- --hide-modules --colors && run-p --race start amp:validate",
-    "test:chromatic": "chromatic test run  --build-script-name build:storybook || true",
+    "test": "npm run test:lint && npm run test:dependencies && npm run test:unit && npm run build -- --hide-modules --colors && run-p --race start amp:validate",
+    "test:chromatic": "chromatic test run --build-script-name build:storybook || true",
```
Comment (jamesbhobbs) [inline package.json]: I think we should remove this, we shouldn't be running it locally anyhow

File: scripts/lighthouseRun.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+npx -p lighthouse@5 lighthouse http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
+npx -p lighthouse@5 lighthouse http://localhost.bbc.com:7080/igbo --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
```
Comment (benjaminhobbs) [inline scripts/lighthouseRun.sh]: why output json and html?

File: package.json [hunk 4]
```diff
@@ -5,54 +5,29 @@
...
+    "cypress:3rdParty": "npm run cypress --project ./3rdPartyCypress/.",
     "cypress:3rdParty:interactive": "npm run cypress:interactive -- --project ./3rdPartyCypress/.",
-    "dataValidate": "node dataValidator/index.js",
-    "dataValidate:debug": "npm run dataValidate --DEBUG_MODE=true",
     "dev": "rm -rf build && cp envConfig/local.env .env && run-p webpack:dev:client webpack:dev:server",
-    "force-package-lock-https": "find . -type f -name \"package-lock.json\" | grep -v node_modules | xargs -I % sh -c 'echo \"Replacing http->https in %\"; perl -pi -e \"s/http:\\/\\//https:\\/\\//g\" %'",
     "killApp": "(lsof -t -i:7080 | xargs kill)",
-    "killStorybook": "(lsof -t -i:9001 | xargs kill)",
-    "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi; \n git update-index --assume-unchanged .env",
-    "speculate": "speculate",
```
Comment (andrew-nowak) [inline package.json:28]: bbc/simorgh-infrastructure#666 still uses `npm run speculate` - you could change the `spec` target in its Makefile to use `npx speculate@2.1.0` instead

File: scripts/lighthouseRun.sh [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```
Comment (andrew-nowak) [inline scripts/lighthouseRun.sh:3]:
```suggestion
set -e

```

Subtle bug - bash scripts won't exit on error by default which means that if the first lighthouse test fails but the second succeeds, the script as a whole would succeed. `set -e` will make the script exit as soon as it finds an error and make the script fail

**Review by andrew-nowak — CHANGES_REQUESTED** [review]

File: scripts/lighthouseRun.sh (hunk 5 above)
Comment (hotinglok) [reply scripts/lighthouseRun.sh:3]: Nice, I didn't know that!
Cheers again

File: package.json (hunk 4 above)
Comment (hotinglok) [reply package.json:28]: Updated.

File: README.md [hunk 6]
```diff
@@ -247,6 +259,15 @@ Here is an example command:
 CYPRESS_APP_ENV=test CYPRESS_UK=true npm run cypress:interactive
 ```
 
+The following command runs both simorgh and cypress:
+
+```
+CYPRESS_APP_ENV=local CYPRESS_UK=true npm run build && npx run-p --race start cypress:interactive
+
```
Comment (andrew-nowak) [inline README.md]:
```suggestion
```

**Review by andrew-nowak — APPROVED** [review]
requested one more change in the simorgh-infra PR, but after that this one lgtm

File: Makefile [hunk 7]
```diff
@@ -9,11 +9,11 @@ install:
...
 	npx apache2-license-checker;
 	npm run test;
-	npm run test:chromatic
+	npx chromatic test run  --build-script-name build:storybook || true
 
 productionTests:
-	npm run build && xvfb-run -a npm run test:prod:ci;
-
+	npm run build && xvfb-run npm run test:ci;
+
```
Comment (dr3) [inline Makefile]:
```suggestion

```

File: Makefile [hunk 8]
```diff
@@ -9,11 +9,10 @@ install:
 developmentTests:
 	npx apache2-license-checker;
 	npm run test;
-	npm run test:chromatic
+	npx chromatic test run  --build-script-name build:storybook || true
 
 productionTests:
-	npm run build && xvfb-run -a npm run test:prod:ci;
-
+	npm run build && xvfb-run npm run test:ci;
```
Comment (andrew-nowak) [inline Makefile]:
```suggestion
	npm run build && xvfb-run -a npm run test:ci;
```
gah merge conflicts

**Review by dr3 — APPROVED** [review]
`revert-3349-revert-3282-revert-3275-revert-3048-AuditScripts` hehe

**Pull request merged**